java-squid/src/main/java/org/sonar/java/model/VisitorsBridge.java

```diff
@@ -116,14 +119,14 @@ public void visitFile(@Nullable AstNode astNode) {
       } else {
         SemanticModel.handleMissingTypes(tree);
       }
-      JavaFileScannerContext context = new DefaultJavaFileScannerContext(tree, (SourceFile) getContext().peekSourceCode(), getContext().getFile(), semanticModel, analyseAccessors);
-      for (JavaFileScanner scanner : scanners) {
-        scanner.scanFile(context);
-      }
-      if (semanticModel != null) {
-        // Close class loader after all the checks.
-        semanticModel.done();
-      }
+    }
+    JavaFileScannerContext context = new DefaultJavaFileScannerContext(tree, (SourceFile) getContext().peekSourceCode(), getContext().getFile(), semanticModel, analyseAccessors);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "context" which hides the field declared at line 70. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)